docs/modules/nifi/pages/troubleshooting/index.adoc

```diff
@@ -86,8 +86,8 @@ nodes:
 
 These properties control NiFi's SNI validation behavior:
 
-* `nifi.web.https.sni.required`: When `true`, NiFi requires that HTTPS requests include an SNI certificate. Setting to `false` allows connections without SNI certificates.
-* `nifi.web.https.sni.host.check`: When `true`, NiFi validates that the SNI certificate's hostname matches the incoming request's hostname. Setting to `false` disables this hostname matching check.
+* `nifi.web.https.sni.required`: When `true`, NiFi requires that HTTPS requests include the SNI extension (containing the `server_name`). Setting to `false` allows connections without SNI certificates.
```

Comment:
We might want to mention that when set to true, hostnames must be used, and connections via IP will fail since the SNI extension isn't used with IP.